geopandas/tests/test_geoseries.py

```diff
@@ -202,7 +203,7 @@ def test_representative_point(self):
     def test_transform(self):
         utm18n = self.landmarks.to_crs(epsg=26918)
         lonlat = utm18n.to_crs(epsg=4326)
-        assert np.all(self.landmarks.geom_almost_equals(lonlat))
+        assert np.all(self.landmarks.geom_equals_exact(lonlat))
```

Comment:
all missing tolerance arg / migrate to use geom_almost_equals from geopandas.tests.util